### PR TITLE
Added entities and repos

### PR DESCRIPTION
**Pull request by ArturVah**

### CODE REVIEW

File: src/TourManager/TourManager.Repository.EfCore/Entities/TourClient.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿namespace TourManager.Repository.EfCore.Entities
+{
+    public class TourClient
+    {
+        public long Id { get; set; }
+
```
Comment (TomaCode) [inline src/TourManager/TourManager.Repository.EfCore/Entities/TourClient.cs:6]: should our ids be typeof long? @HAghajanyan-ST

File: src/TourManager/TourManager.Api/appsettings.json [hunk 2]
```diff
@@ -6,5 +6,9 @@
       "Microsoft.Hosting.Lifetime": "Information"
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+
+  "ConnectionStrings": {
+    "DefaultConnection": "Provider=MSOLEDBSQL;Server=localhost:3306;Initial Catalog=tourdb;MultipleActiveResultSets=true;User ID=root;Password=root"
```
Comment (HAghajanyan-ST) [inline src/TourManager/TourManager.Api/appsettings.json:12]: Please fix connection string

File: src/TourManager/TourManager.Api/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+version: '3.1'
+
+services:
+
+  db:
+    image: mcr.microsoft.com/mssql/server:2017-latest
```
Comment (HAghajanyan-ST) [inline src/TourManager/TourManager.Api/docker-compose.yml:6]: Did you try this setup ?

File: src/TourManager/TourManager.Repository.Abstraction/TourManager.Repository.Abstraction.csproj [hunk 4]
```diff
@@ -1,7 +1,15 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
         <TargetFramework>net5.0</TargetFramework>
     </PropertyGroup>
 
+    <ItemGroup>
+      <PackageReference Include="Microsoft.EntityFrameworkCore" Version="5.0.2" />
```
Comment (HAghajanyan-ST) [inline src/TourManager/TourManager.Repository.Abstraction/TourManager.Repository.Abstraction.csproj:8]: Remove reference

File: src/TourManager/TourManager.Repository.Abstraction/TourManager.Repository.Abstraction.csproj [hunk 5]
```diff
@@ -1,7 +1,15 @@
...
+    </ItemGroup>
+
+    <ItemGroup>
+      <ProjectReference Include="..\TourManager.Repository.EfCore\TourManager.Repository.EfCore.csproj" />
```
Comment (HAghajanyan-ST) [inline src/TourManager/TourManager.Repository.Abstraction/TourManager.Repository.Abstraction.csproj:12]: Remove reference

**Review by HAghajanyan-ST — CHANGES_REQUESTED** [review]

File: src/TourManager/TourManager.Repository.EfCore/Entities/TourClient.cs (hunk 1 above)
Comment (HAghajanyan-ST) [reply src/TourManager/TourManager.Repository.EfCore/Entities/TourClient.cs:6]: Yes Tom jan, I think it's ok, but you are right it can be `int` as well .

**Pull request closed without merging**